### PR TITLE
listQueries displays startime in UTC

### PR DESCRIPTION
**Pull request by praveenag**

### CODE REVIEW

**Review by thobe — APPROVED** [review]
Looks good.

**Review by spacecowboy — APPROVED** [review]

File: enterprise/kernel/src/main/java/org/neo4j/kernel/enterprise/builtinprocs/EnterpriseBuiltInDbmsProcedures.java [hunk 1]
```diff
@@ -481,8 +481,8 @@ private QueryStatusResult queryStatusResult( ExecutingQuery q )
         private static String formatTime( final long startTime )
         {
             return OffsetDateTime
-                .ofInstant( Instant.ofEpochMilli( startTime ), ZoneId.systemDefault() )
-                .format( ISO_OFFSET_DATE_TIME );
+                    .ofInstant( Instant.ofEpochMilli( startTime ), ZoneId.of("GMT") )
```
Comment (pontusmelke) [inline enterprise/kernel/src/main/java/org/neo4j/kernel/enterprise/builtinprocs/EnterpriseBuiltInDbmsProcedures.java]: It doesn't really matter but shouldn't it be `UTC`?

Comment (spacecowboy) [reply enterprise/kernel/src/main/java/org/neo4j/kernel/enterprise/builtinprocs/EnterpriseBuiltInDbmsProcedures.java]: Hmm, good point

Comment (chrisvest) [reply enterprise/kernel/src/main/java/org/neo4j/kernel/enterprise/builtinprocs/EnterpriseBuiltInDbmsProcedures.java]: Also, how about lifting the `ZoneId` to a constant? Those instances are immutable and thread-safe.

**Review by chrisvest — COMMENTED** [review]
Reviewed. I agree with Pontus, and had one additional comment.

**Pull request merged**